farm/tests/src/psp22/psp22_contract.rs

```diff
@@ -0,0 +1,267 @@
+use scale::Encode as _;
```

Comment:
This file is auto-generated. Shouldn't it be in `.gitignore`?

Reply:
It's auto-generated but the contracts used for generating it are hardcoded into the project (`psp22` and `wrapped_azero`) so it doesn't need to be re-generated.